File: Vanilla/sources/zdoc/media/lua/shared/Library/ZombieSittingState.lua
---@class ZombieSittingState : zombie.ai.states.ZombieSittingState
---@field private _instance ZombieSittingState
ZombieSittingState = {}

---@public
---@param arg0 IsoGameCharacter
---@return void
function ZombieSittingState:enter(arg0) end

---@public
---@return ZombieSittingState
function ZombieSittingState:instance() end

---@public
---@param arg0 IsoGameCharacter
---@return void
function ZombieSittingState:execute(arg0) end

---@public
---@param arg0 IsoGameCharacter
---@return void
function ZombieSittingState:exit(arg0) end

File: Vanilla/sources/zomboid/zombie/ai/states/ZombieSittingState.java
package zombie.ai.states;

import zombie.ai.State;
import zombie.characters.IsoGameCharacter;

public final class ZombieSittingState extends State {
   private static final ZombieSittingState _instance = new ZombieSittingState();

   public static ZombieSittingState instance() {
      return _instance;
   }

   public void enter(IsoGameCharacter var1) {
   }

   public void execute(IsoGameCharacter var1) {
   }

   public void exit(IsoGameCharacter var1) {
   }
}
